src/components/Checkout/DonationNudge/DonationNudgeBoxes.vue

```diff
@@ -250,6 +264,20 @@ export default {
 		};
 	},
 	methods: {
+		formatAndSubmitOnEnter() {
+			const customInput = document.getElementById('customDonationInput');
+			const customInputButton = document.querySelector('.custom-amount-submit');
+
+			customInput.addEventListener('keyup', event => {
+				if (event.keyCode === 13) {
+					customInput.blur();
+					customInputButton.click();
+				}
+			});
```

Comment:
I think everything here could go away and you could just call setCustomInputAndClose... may even be able to call that directly from the @click.enter="setCustomInputAndClose"

Reply:
if this would require a significant refactor of the component you can skip for now...